tutorials/turtlebot4_navigator.md

```diff
@@ -1300,4 +1300,317 @@ else:
 </figure>
 
 {% endtab %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+## LLM Integration
+
+This example demonstrates an integration between the OpenAI [Chat Completions API](https://platform.openai.com/docs/guides/gpt/chat-completions-api) and the Turtlebot 4, substantially inspired by the ['Code as Policies: Language Model Programs for Embodied Control' (Liang et al.)](https://arxiv.org/abs/2209.07753) research.
+
+It demonstrates how an LLM can be 'taught' how to use the Turtlebot 4 Navigation API with only a few examples, and also shows how an LLM can easily bridge natural language commands to API calls without an intermediate parsing step.
```

Comment:
Can we clarify `Large Language Model (LLM)` here once on the page (for anyone not familiar with the acronym)

Reply:
Sure!